pinot-controller/src/test/java/com/linkedin/pinot/controller/helix/ControllerTenantTest.java

```diff
@@ -15,297 +15,158 @@
-      LOGGER.trace("**************");
-      LOGGER.trace(res);
-      resJsonObject = new JSONObject(res);
-      Assert.assertEquals(resJsonObject.getJSONArray("BrokerInstances").length(), 5);
-      Assert.assertEquals(resJsonObject.getJSONArray("ServerInstances").length(), 0);
-      Assert.assertEquals(resJsonObject.getString("tenantName"), brokerTag);
+    // Get server tenants
+    JSONObject response = new JSONObject(sendGetRequest(_requestBuilder.forTenantGet()));
+    Assert.assertEquals(response.getJSONArray("SERVER_TENANTS").length(), 2);
+    for (int i = 0; i < 2; i++) {
+      String serverTag = SERVER_TAG_PREFIX + i;
+      response = new JSONObject(sendGetRequest(_requestBuilder.forServerTenantGet(serverTag)));
+      Assert.assertEquals(response.getJSONArray("ServerInstances").length(), 3);
+      Assert.assertEquals(response.getString("tenantName"), serverTag);
+      response = new JSONObject(sendGetRequest(_requestBuilder.forTenantGet(serverTag)));
+      Assert.assertEquals(response.getJSONArray("BrokerInstances").length(), 0);
+      Assert.assertEquals(response.getJSONArray("ServerInstances").length(), 3);
+      Assert.assertEquals(response.getString("tenantName"), serverTag);
     }
-    for (int i = 0; i < 4; i++) {
-      String brokerTag = "colocated_" + i;
-      res =
-          sendDeleteRequest(ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).forBrokerTenantDelete(
-              brokerTag));
-      LOGGER.trace(res);
-      Assert
-          .assertEquals(_helixAdmin.getInstancesInClusterWithTag(HELIX_CLUSTER_NAME, brokerTag + "_BROKER").size(), 0);
-      Assert.assertEquals(
-          _helixAdmin.getInstancesInClusterWithTag(HELIX_CLUSTER_NAME, CommonConstants.Helix.UNTAGGED_BROKER_INSTANCE)
-              .size(), i * 5 + 5);
-    }
-  }
 
-  @Test
-  public void testGetServerCall() throws JSONException, UnsupportedEncodingException, IOException {
-    for (int i = 0; i < 4; i++) {
-      String serverTag = "serverTag_" + i;
-      final JSONObject payload = ControllerRequestBuilderUtil.buildServerTenantCreateRequestJSON(serverTag, 5, 2, 3);
-      final String res =
-          sendPostRequest(ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).forTenantCreate(),
-              payload.toString());
-      LOGGER.trace(res);
-      Assert.assertEquals(_helixAdmin.getInstancesInClusterWithTag(HELIX_CLUSTER_NAME, serverTag + "_OFFLINE").size(),
-          2);
-      Assert.assertEquals(_helixAdmin.getInstancesInClusterWithTag(HELIX_CLUSTER_NAME, serverTag + "_REALTIME").size(),
-          3);
+    // Update server tenants
+    // Note: server tenants cannot scale down
+    for (int i = 0; i < 3; i++) {
+      String serverTag = SERVER_TAG_PREFIX + 0;
+      JSONObject payload =
+          ControllerRequestBuilderUtil.buildServerTenantCreateRequestJSON(serverTag, 3 + 2 * i, 2 + i, 1 + i);
```

Comment:
Lets try to avoid the magic math that is a function of some other constant somewhere else (10 in this case). Reason being, if we want to change the original constant (10 in this case), we need to re-do the math here.